.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,12 +8,12 @@ run the test suite, write new integrations, and more.
 **2.0 Upgrade Guide notes**
 <!--
 (If this PR is for 1.x, please delete this section)
-A public facing description that will go into the [upgrade guide](https://github.com/DataDog/dd-trace-rb/blob/master/docs/UpgradeGuide.md) for this change.
-We already know what the PR does (from the PR title),
-but users should know either:
+If this PR introduces a breaking change, update the
```

Comment:
should we make it explicit that it be part of this PR and not a separate one?

Reply:
If it's not clear for you, it's not clear for me!